modules/cassandra/options.go

```diff
@@ -0,0 +1,127 @@
+package cassandra
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/testcontainers/testcontainers-go"
+)
+
+// TLSConfig represents the TLS configuration for Cassandra
+type TLSConfig struct {
+	KeystorePath    string
+	CertificatePath string
+	Config          *tls.Config
+}
+
+// Options represents the configuration options for the Cassandra container
+type Options struct {
+	tlsConfig *TLSConfig
+}
+
+// Option is an option for the Cassandra container.
+type Option func(*testcontainers.GenericContainerRequest, *Options) error
+
+// Customize implements the testcontainers.ContainerCustomizer interface
+func (o Option) Customize(req *testcontainers.GenericContainerRequest) error {
+	return o(req, &Options{})
```

Comment:
bug: while this will run there options won't be accessible, so should still be just a noop.

We're looking to refactor how this works in a future version to avoid this, but that will be a bit.